commune/settings.py

```diff
@@ -30,6 +30,7 @@
 
 # Application definition
```

Comment:
this might as well be a python docstring

https://www.programiz.com/python-programming/docstrings

see the section on docstrings for python modules